exercises/custom-set/example.rb

```diff
@@ -36,8 +40,9 @@ def intersection(other)
   def member?(datum)
     data.any? { |node| node.datum.eql?(datum) }
   end
+  alias include? member?
```

Comment:
Enumerable aliases `member?` to `include?` while internally, it uses `member`.  Standard library `Set` has Ruby method `include?` which it aliases to `member?`.